### PR TITLE
Update websites dependencies

### PR DESCRIPTION
**Pull request by jpraynaud**
## Content
This PR includes update of the dependencies of the repository websites with patch of vulnerabilities:
- doc website
- explorer websites
- WASM client websites

## Pre-submit checklist

- Branch
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested

### CODE REVIEW

**Review by dlachaume — APPROVED** [review]
LGTM 🚀

File: test.sh
Comment (sfauvel) [inline test.sh:1]: We may rename this script and explain when and why we have to execute it

**Review by sfauvel — CHANGES_REQUESTED** [review]
File `test.sh` should be renamed

**Comment by jpraynaud** [comment]
> File `test.sh` should be renamed

This file has been added to the commit by mistake. I removed it

**Review by Alenar — APPROVED** [review]

**Review by sfauvel — APPROVED** [review]

**Pull request merged**